### PR TITLE
Updated cryostream signal for AMX

### PR DESCRIPTION
**Pull request by vshekar**
Set up cryostream temp value to the GUI

### CODE REVIEW

File: gui/control_main.py [hunk 1]
```diff
@@ -142,7 +142,7 @@ class ControlMain(QtWidgets.QMainWindow):
     roiChangeSignal = QtCore.Signal(int, str)
     highMagCursorChangeSignal = QtCore.Signal(int, str)
     lowMagCursorChangeSignal = QtCore.Signal(int, str)
-    cryostreamTempSignal = QtCore.Signal(str)
+    cryostreamTempSignal = QtCore.Signal(object)
```
Comment (JunAishima) [inline gui/control_main.py:145]: is there any reason why it couldn't be a float or something?

File: config_params.py [hunk 2]
```diff
@@ -83,7 +83,7 @@ class RasterStatus(Enum):
 DEWAR_SECTORS = {"amx": 8, "fmx": 8, "nyx": 5}
 PUCKS_PER_DEWAR_SECTOR = {"amx": 3, "fmx": 3, "nyx": 3}
 
-cryostreamTempPV = {"amx": "AMX:cs700:gasT-I", "fmx": "FMX:cs700:gasT-I"}
+cryostreamTempPV = {"amx": "XF:17IDB-ES:AMX{CS:1}SAMPLE_TEMP_RBV", "fmx": "FMX:cs700:gasT-I"}
```
Comment (JunAishima) [inline config_params.py:86]: FMX's PB does not appear to be working right now (March 8, 10:23am) - can this be updated now?

**Review by JunAishima — CHANGES_REQUESTED** [review]

File: config_params.py (hunk 2 above)
Comment (JunAishima) [reply config_params.py:86]: any update on the actual PV that works for FMX?

Comment (vshekar) [reply config_params.py:86]: No, FMX does not have a working system, they are going to install it during the upcoming shutdown in which case I will update this PV

File: gui/control_main.py (hunk 1 above)
Comment (vshekar) [reply gui/control_main.py:145]: For FMX currently the value returned by the PV is None whereas its a float at AMX. This is a catch all for different values. It is correctly converted to string in the GUI. Once the FMX PV provides correct values we can switch it to float

File: config_params.py (hunk 2 above)
Comment (JunAishima) [reply config_params.py:86]: good enough for now then...

**Review by JunAishima — APPROVED** [review]

**Comment by JunAishima** [comment]
created this issue for future reference, about the cryostream PV: https://github.com/NSLS-II/lsdc/issues/374

**Pull request merged**